feat(cluster): clarify requirements when scaling down (#52)
anbox-cloud/discourse-docs: howto/cluster/scale-down.md

Scaling down a LXD cluster involves more checks than scaling up.

> There are two important things to know when scaling down:
>  - The node you remove must not have any container left.
>  - You must wait for a node to be fully removed before you can start removing another one

Reviewer: ```suggestion
There are two important requirements when scaling down:
 - The node you remove must not have any containers left.
 - You must wait for a node to be fully removed before you can start removing another one.
```
There were a few more changes in here. If you prefer your intro that's fine with me, but I'll insist on the plural and the period at the end. ;)

## Prepare the node for removal

First, pick the node you want to remove and tell AMS to stop considering this node for new containers:

    amc node set lxd0 unscheduable true

(Note that the typo in the command will be fixed in a future Anbox Cloud release.)

Now the node won't be considered for any further container launches.

The node might still have containers running. You can decide to either kill all containers or wait for your users to disconnect.

> Use the following command if the node still hosts containers:

Reviewer: ```suggestion
Use the following command to check whether the node still hosts containers:
```
This is for checking, not for removing - correct?

Author: Correct !

Scaling down a LXD cluster involves more checks than scaling up.

> There are two important things to know when scaling down:
>  - The node you remove **MUST NOT** have any container left.
>  - You **MUST** wait for a node to be fully removed before you can start removing another one

Reviewer: ```suggestion
There are two important requirements when scaling down:
 - The node you remove must not have any containers left.
 - You must wait for a node to be fully removed before you can start removing another one.
```

> 1. Preparing the node for removal

Reviewer: ```suggestion
## Prepare the node for removal
```
Since these steps are quite long, let's turn them into sections. Not sure if we should keep the numbers - I'm okay with keeping them, but I don't think they are really needed. But up to you :)

> First, pick the node you want to remove.
> We have to tell AMS to stop considering this node for new containers.

Reviewer: ```suggestion
First, pick the node you want to remove and tell AMS to stop considering this node for new containers.
```
(I don't like the "we" in there since we don't have it in the rest of the document.)

You can mark a node as unschedulable with the following command:

    amc node set lxd0 unscheduable true

(Note that the typo in the command will be fixed in a future Anbox Cloud release.)

Now the node won't be considered for any further container launches.

> The node might still have containers running, you can decide to either kill all containers or wait for your users to disconnect. You can check with the following command if the node still hosts containers:

Reviewer: ```suggestion
The node might still have containers running. You can decide to either kill all containers or wait for your users to disconnect. Use the following command to check whether the node still hosts containers:
```

    amc ls --filter node=lxd0 --format=csv | wc -l

Note that a node **MUST** have no container left on it before you can remove it.

If you want to kill all containers immediately, run the following command:

    amc delete --all --yes

> 2. Removing the node

Reviewer: ```suggestion
## Remove the node
```

When the node is ready to be removed, use the following Juju command:

    juju remove-unit lxd/<id>

> where `<id>` is the number of the unit you are removing

Reviewer: Move this to the intro ("When the node is ready to be removed, use the following Juju command, where `<id>` is the number of the unit you are removing:")
